### PR TITLE
Tests: Dockerfile for Nginx for swarm tests

### LINKED ISSUE #1014: Create own nginx image to not stick with old nginx:1.13.0
Our swarm tests work with the `nginx:1.13.0` image. But we cannot stick forever to that tag. 
The best solution at the moment would be to create our own image, based on the upstream nginx image.

### PR DESCRIPTION
**Pull request by GabyCT**
This dockerfile will build an image for nginx for the swarm tests
that we currently have in clear containers.

Fixes #1014

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by gorozco1** [comment]
why this path `cc-oci-runtime/tests/integration/docker/Dockerfiles/` 

Wouldn't make sense to create it under

`cc-oci-runtime/tests/integration/Dockerfiles/nginx`

**Comment by GabyCT** [comment]
@gorozco1, I put it there because is only used for docker tests

**Comment by gorozco1** [comment]
lgtm makes sense.

**Comment by chavafg** [comment]
qa-passed

**Comment by jcvenegas** [comment]
@GabyCT  hey seems that ubuntu servers are failing

```
E: Failed to fetch https://debian.neo4j.org/repo/stable/Packages.gz  Hash Sum mismatch
E: Some index files failed to download. They have been ignored, or old ones used instead.
```

**Comment by chavafg** [comment]
qa-passed

**Comment by jodh-intel** [comment]
I agree with @gorozco1 - it would still be helpful to indicate what this particular `Dockerfile` is for.

How about calling this  `Dockerfile.nginx` or `Dockerfile.swarm` for clarity? I'd add a brief `README.md` too explaining:

- what the file is for.
- how to use it (`docker build -t $name -f Dockerfile.swarm .`).

**Comment by GabyCT** [comment]
@jodh-intel changes were applied

**Comment by jodh-intel** [comment]
lgtm.

@iphutch - could you take a look?

**Comment by chavafg** [comment]
qa-passed

**Comment by chavafg** [comment]
qa-passed

File: tests/integration/docker/Dockerfiles/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# NGINX image for Intel® Clear Containers swarm tests
+
+In order to build the nginx image which is used in our swarm tests,
+follow these steps:
+
+1. Building the nginx image with the following command:
```
Comment (iphutch) [inline tests/integration/docker/Dockerfiles/README.md]: nit: replace Building with Build
Build the nginx image with the following command:

Comment (devimc) [inline tests/integration/docker/Dockerfiles/README.md]: or ```To build``` ?

**Comment by GabyCT** [comment]
@iphutch changes applied

**Comment by chavafg** [comment]
qa-passed

**Comment by devimc** [comment]
lgtm

**Comment by devimc** [comment]
Hi @iphutch 
could you review it?

**Comment by grahamwhaley** [comment]
Ooh, looks like we hit the Travis SHA issue @GabyCT - trying a rebase or small mode to get a SHA change and then a push quite often fixes it. Sigh, might mean we need to gather racks again though. grumble...

**Comment by chavafg** [comment]
qa-passed

**Pull request merged**